src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -414,7 +421,19 @@ private TypeWithAnnotations ComputeReturnType(DiagnosticBag diagnostics)
             else
             {
                 var binder = GetBinder();
-                return TypeWithAnnotations.Create(binder.GetSpecialType(SpecialType.System_Void, diagnostics, this.GetSyntax()));
+                var type = TypeWithAnnotations.Create(binder.GetSpecialType(SpecialType.System_Void, diagnostics, this.GetSyntax()));
+
+                if (IsInitOnly)
+                {
+                    var isInitOnlyType = Binder.GetWellKnownType(this.DeclaringCompilation,
```

Comment:
Shouldn't we synthesize the type if it doesn't exist, instead of reporting an error?

Reply:
That was only in an early draft of the spec, it was updated since. The type must exist.

---
In reply to: [413377890](https://github.com/dotnet/roslyn/pull/43275#discussion_r413377890) [](ancestors = 413377890)

Reply:
> Shouldn't we synthesize the type if it doesn't exist, instead of reporting an error?

We do not synthesize types for custom modifiers because we would have to do that too early

---
In reply to: [413407600](https://github.com/dotnet/roslyn/pull/43275#discussion_r413407600) [](ancestors = 413407600,413377890)